core/storage/blog/gae_models.py

```diff
@@ -334,16 +334,25 @@ def has_reference_to_user_id(cls, user_id):
             cls.editor_ids == user_id).get(keys_only=True) is not None
 
     @classmethod
-    def get_by_user(cls, user_id):
+    def get_by_user(cls, user_id, published=False, limit=None):
         """Retrieves the rights object for all blog posts assigned to given user
 
         Args:
             user_id: str. ID of user.
```

Comment:
Describe more who the user is from this perspective (e.g. ID of the user whose blog posts are to be fetched).

Reply:
Done.